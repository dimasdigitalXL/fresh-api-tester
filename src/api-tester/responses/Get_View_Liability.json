{
  "error": {
    "code": 7431,
    "http_code": 404,
    "message": "Route not found",
    "href": "https://636a1d639f14e.xentral.biz/api/v1/liabilities/%7Bid%7D/docs.html#error-7431"
  }
}
