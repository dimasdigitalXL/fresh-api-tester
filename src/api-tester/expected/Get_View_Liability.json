{
  "data": {
    "id": "string",
    "documentNumber": "string",
    "date": "string",
    "createdAt": "string",
    "createdBy": {
      "name": "string"
    },
    "updatedAt": "string",
    "status": "string",
    "supplier": {
      "id": "string"
    },
    "supplierInvoiceNumber": "string",
    "receiptDate": "string",
    "serviceDate": null,
    "financials": {
      "amountGross": {
        "amount": "string",
        "currency": "string"
      },
      "amountNet": {
        "amount": "string",
        "currency": "string"
      },
      "freightCharge": 0,
      "taxation": "string",
      "taxes": {
        "normalTax": {
          "rate": 0,
          "amount": 0
        },
        "reducedTax": {
          "rate": 0,
          "amount": 0
        },
        "customTax1": {
          "rate": 0,
          "amount": 0
        },
        "customTax2": {
          "rate": 0,
          "amount": 0
        }
      },
      "paymentTerms": {
        "dueDate": "string",
        "earlyPaymentDate": "string",
        "earlyPaymentDiscount": 0,
        "earlyPaymentDiscountAmount": 0
      },
      "bookingText": "string",
      "payment": {
        "methodName": "string",
        "paidAmount": 0,
        "paidOnDate": "string",
        "discountReceived": 0
      },
      "costCenter": "string",
      "glAccount": "string"
    },
    "purchaseOrder": null,
    "assignedPurchaseOrders": [],
    "project": {
      "id": "string"
    },
    "accounting": [
      {
        "glAccount": "string",
        "bookingText": "string",
        "amount": {
          "amount": "string",
          "currency": "string"
        },
        "tax": {
          "rate": 0
        },
        "costCenter": "string"
      }
    ],
    "positions": [],
    "purpose": null
  }
}
